distribution/lib/Standard/AWS/0.0.0-dev/src/S3/S3.enso

```diff
@@ -7,7 +7,8 @@ import project.Internal.Auth
 polyglot java import software.amazon.awssdk.services.s3.model.S3Exception
 polyglot java import software.amazon.awssdk.services.s3.S3Client
 
-## Gets the list of the S3 bucket names.
+## GROUP Standard.Base.Input
```

Comment:
Please update the _Tags_ section of the doc comments https://github.com/enso-org/enso/blob/d7e5c3f36274ab57edec649cec9578108ccaa31d/docs/syntax/comments.md#tags